application/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(application VERSION 0.0.1 LANGUAGES)
```

Comment:
Something after languages? Is `languages` necessary?

Reply:
It's optional for C and CPP. Will remove it.